src/test/java/org/isf/patient/rest/PatientControllerTest.java

```diff
@@ -61,18 +64,21 @@
 	@Mock
     private PatientBrowserManager patientBrowserManagerMock;
 	
-	@Autowired
-	private static PatientMapper mapper;
-	
+	private static PatientMapper mapper = new PatientMapper();
```

Comment:
Naming this field `patientMapper` as well as removing the `static` would be clearer.

`private PatientMapper patientMapper = new PatientMapper();`

Then you could fix  `PatientDTOHelper` `setup` method like this:

```
		public static PatientDTO setup(PatientMapper patientMapper) throws OHException{
			Patient patient = PatientHelper.setupPatient();
			return patientMapper.map2DTO(patient);
		}
```
and provide `patientMapper` for all the calls of `PatientDTOHelper` `setup` method. 

Note: that in a different pull request I've already moved the helper classes to independent files, having an static field to be able to use the object in a nested class in a single place is not enough reason to add coupling.

Reply:
Yes I agree.
Thanks!